package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@woodwing/csde-components-validator",
-  "version": "1.6.0",
+  "version": "1.6.1",
```

Comment:
We don't really need to publish a new version for this.

Reply:
yeah, right